### PR TITLE
Improve the environment specific phase banners

### PR DESCRIPTION
**Pull request by peteryates**
### Context

Now they will have a colour per environment and tests cover the env-specific text

### Previews

| Environment | Preview |
| ------------ | ------- |
| Development | Screenshot from 2023-07-28 11-05-39 |
| Review | image |
| Staging | Screenshot from 2023-07-28 11-05-16 |
| Sandbox 🏖️ | Screenshot from 2023-07-28 11-04-52 |
| Production | Screenshot from 2023-07-28 11-05-58 |

### CODE REVIEW

File: app/helpers/phase_banner_helper.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+module PhaseBannerHelper
+  def phase_banner_tag_text(env = Rails.env)
+    return "Beta" if env == "production"
+
+    env
+  end
+
+  def phase_banner_tag_colour(env = Rails.env)
+    {
+      "development" => "red",
```
Comment (cwrw) [inline app/helpers/phase_banner_helper.rb:12]: could we include deployed development?

Comment (cwrw) [reply app/helpers/phase_banner_helper.rb:12]: maybe also review?

Comment (peteryates) [reply app/helpers/phase_banner_helper.rb:12]: oh review, good call. Was going to omit deployed_development as it'll be short-lived

Comment (peteryates) [reply app/helpers/phase_banner_helper.rb:12]: colour nominations for review?

Comment (peteryates) [reply app/helpers/phase_banner_helper.rb:12]: it's purple, for now!

**Review by cwrw — APPROVED** [review]

**Pull request merged**